cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/GetController.java

```diff
@@ -0,0 +1,87 @@
+package cloud.foundry.cli.services;
+
+import cloud.foundry.cli.crosscutting.beans.ApplicationBean;
+import cloud.foundry.cli.crosscutting.beans.ServiceInstanceSummaryBean;
+import cloud.foundry.cli.crosscutting.beans.SpaceDevelopersBean;
+import cloud.foundry.cli.crosscutting.mapping.CfOperationsCreator;
+import cloud.foundry.cli.crosscutting.util.YamlUtils;
+import cloud.foundry.cli.operations.ApplicationOperations;
+import cloud.foundry.cli.operations.ServicesOperations;
+import cloud.foundry.cli.operations.SpaceDevelopersOperations;
+import org.cloudfoundry.operations.DefaultCloudFoundryOperations;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Mixin;
+
+import java.util.List;
+
+/**
+ * Controller for the Get-Commands
+ */
+@Command(name = "get",
+        header = "%n@|green Get-Controller|@",
+        subcommands = {
+                GetController.GetServicesCommand.class,
+                GetController.GetSpaceDevelopersCommand.class,
+                GetController.GetApplicationsCommand.class})
+public class GetController implements Runnable {
+
+    @Override
+    public void run() {
+        //getSpaceDevelopers
+        //getServices
+        //getApps
+        // construct yaml
+    }
+
+    @Command(name = "space-developers",
+            description = "List all space developers in the target space")
+    static class GetSpaceDevelopersCommand implements Runnable {
+        @Mixin
+        GetControllerCommandOptions commandOptions;
+
+        @Override
+        public void run() {
```

Comment:
I already mentioned that there's some duplicate code in all these `Runnable`s. We should really add an abstract base class to fix that.